bin/ds

```diff
@@ -471,7 +471,7 @@ function pull_db {
       drush -y fra
 
       echo "Disabling Optimizely module on staging."
-      drush dis optimizely
+      drush dis optimizely -y
```

Comment:
It will be more consistent to pass `-y` as the first argument: this is core-global-option. Not all drush submodules accept global options passed passed as the first argument.